luda-editor/luda-editor-client/src/app/editor/mod.rs

```diff
@@ -204,8 +201,7 @@ impl namui::Entity for Editor {
 }
 
 impl Editor {
-    fn new(screen_wh: namui::Wh<f32>) -> Self {
-        let socket = Editor::create_socket();
+    pub fn new(screen_wh: namui::Wh<f32>, socket: Socket, sequence: Sequence) -> Self {
```

Comment:
I `think screen_wh` should not be a state of Editor, but just a props. I will make it clear what kind of data should be in entity struct and props later.

Reply:
Done with 4984bf0